packages/gatsby-theme-carbon/src/components/markdown/Markdown.module.scss

```diff
@@ -93,6 +93,26 @@
   color: $carbon--white-0;
 }
 
+//spacing for content within list item
+.list-item > * + :global(.bx--row),
+.list-item > * + :global(.gatsby-resp-image-wrapper),
+.list-item > * + pre {
+  margin-bottom: 1.5rem;
+}
+
+//second child in li needs top margin to separate it from li text
+.list-item > :global(.bx--row):nth-child(2),
```

Comment:
Definitely appreciate the comments, makes really clear what the intent is.

I think taking the nested variant approach will work out best here: https://every-layout.dev/layouts/stack/#nested-variants

Another thing to keep in mind (might not be helpful here) is using dot notation in our MDX provider to scope/target the list item component https://www.christopherbiscardi.com/devtips/mdx